pkg/controller/core.oam.dev/v1alpha2/applicationrollout/helper.go

```diff
@@ -49,17 +49,27 @@ func RolloutWorkloadName(rolloutComp string) assemble.WorkloadOption {
 
 		// we hard code the behavior depends on the workload group/kind for now. The only in-place upgradable resources
 		// we support is cloneset/statefulset for now. We can easily add more later.
+		supportInplaceUpgrade := false
 		if w.GroupVersionKind().Group == v1alpha1.GroupVersion.Group {
 			if w.GetKind() == reflect.TypeOf(v1alpha1.CloneSet{}).Name() ||
 				w.GetKind() == reflect.TypeOf(v1alpha1.StatefulSet{}).Name() {
```

Comment:
This line is useless, please delete.

Reply:
The `supportInplaceUpgrade` variable will be used to determine if the workload supports in-place upgrade down below:
```go
if supportInplaceUpgrade {
	// we use the component name alone for those resources that do support in-place upgrade
	klog.InfoS("we reuse the component name for resources that support in-place upgrade",
		"GVK", w.GroupVersionKind(), "instance name", w.GetName())
	// assemble use component name as workload name by default
	// so no need to re-set name
	return nil
}
```
Should we keep this part?

Reply:
I mean only line 55 is useless.